src/Cli/dotnet/commands/dotnet-tool/update/ToolUpdateCommandParser.cs

```diff
@@ -10,27 +10,14 @@ namespace Microsoft.DotNet.Cli
 {
     internal static class ToolUpdateCommandParser
     {
-        public static readonly CliArgument<string> PackageIdArgument = ToolInstallCommandParser.PackageIdArgument;
-
-        public static readonly CliOption<bool> GlobalOption = ToolAppliedOption.GlobalOption;
-
-        public static readonly CliOption<string> ToolPathOption = ToolAppliedOption.ToolPathOption;
-
-        public static readonly CliOption<bool> LocalOption = ToolAppliedOption.LocalOption;
-
-        public static readonly CliOption<string> ConfigOption = ToolInstallCommandParser.ConfigOption;
-
-        public static readonly CliOption<string[]> AddSourceOption = ToolInstallCommandParser.AddSourceOption;
-
-        public static readonly CliOption<string> FrameworkOption = ToolInstallCommandParser.FrameworkOption;
-
-        public static readonly CliOption<string> VersionOption = ToolInstallCommandParser.VersionOption;
-
-        public static readonly CliOption<string> ToolManifestOption = ToolAppliedOption.ToolManifestOption;
-
-        public static readonly CliOption<bool> PrereleaseOption = ToolSearchCommandParser.PrereleaseOption;
+        public static readonly CliArgument<string> PackageIdArgument = new("packageId")
+        {
+            HelpName = LocalizableStrings.PackageIdArgumentName,
+            Description = LocalizableStrings.PackageIdArgumentDescription,
+            Arity = ArgumentArity.ZeroOrOne
+        };
 
-        public static readonly CliOption<VerbosityOptions> VerbosityOption = ToolInstallCommandParser.VerbosityOption;
+        public static readonly CliOption<bool> UpdateAllOption = ToolAppliedOption.UpdateAllOption;
```

Comment:
Thanks for the consolidation of `UpdateAllOption` in `ToolAppliedOption`. I personally think it's a great idea since the `-all` option might applies to other command as well.